build.gradle

```diff
@@ -190,15 +191,17 @@ subprojects {
 			}
 		}
 	}
-	signing {
-		sign publishing.publications.mavenJava
-	}
-	publishMavenJavaPublicationToMavenRepository.doFirst {
-		if(OSSRH_URL == null) {
-			throw new GradleException("Please specify the variable 'ossrhUrl' in '~/.gradle/gradle.properties'.")
+	if(SIGNING_KEY_DEFINED) {
```

Comment:
hmm, dropping signing like this could lead to ppl forgetting it and then we publish without singing erroneously